.github/workflows/swift-toolchain.yml

```diff
@@ -25,6 +25,33 @@ on:
         description: 'Code Sign'
         default: false
         type: boolean
+  workflow_call:
+    inputs:
+      snapshot:
+        description: 'Build Swift at a tagged snapshot'
+        default: false
+        type: boolean
```

Comment:
Hmm, we really should see if it is possible to build at a tagged snapshot at this point. I think that with the context step, it no longer is, and it might make sense to just drop this option. But that can be a separate change.